src/url-loader/fetch-url-loader.ts

```diff
@@ -46,4 +46,8 @@ export class FetchUrlLoader implements UrlLoader {
       }
     });
   }
+
+  listFilesInProject(): Promise<string[]> {
+    throw new Error('Not implementable.');
```

Comment:
let's do: "Not supported", since this could be implemented in a subclass with a known server protocol.